src/cipher/encryptingstorage/tests/test_zodb.py

```diff
@@ -476,7 +539,9 @@ def test_suite():
             'encryptingstoragetests.%s' % class_.__name__)
         suite.addTest(s)
 
+    suite.addTest(unittest.makeSuite(TestIterator))
+    suite.addTest(unittest.makeSuite(TestServerEncryptingStorage))
     suite.addTest(doctest.DocTestSuite(
-        setUp=setupstack.setUpDirectory, tearDown=setupstack.tearDown
+        setUp=setupstack.setUpDirectory, tearDown=ZODB.tests.util.tearDown
```

Comment:
I don't understand this change.  What is setupstack?  Will ZODB.tests.util.tearDown call setuptstack.tearDown?

Reply:
Yes, and clear data managers registered with the global transaction manager. I saw the change in ZODB 5 tests, so why not use it here too.